### PR TITLE
[luci] Support quantization of LogicalOr

### PR DESCRIPTION
**Pull request by jinevening**
This supports quantization of LogicalOr op.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/6367

### CODE REVIEW

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -580,7 +560,7 @@ struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<bool>
         continue;
 
       // Check if this is bool type (bool type is not quantized)
-      if (bool_type(circle_node))
+      if (circle_node->dtype() == loco::DataType::BOOL)
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:563]: this change doesn't seem to be related with `LogicalOr`. It would be better to post in anothr PR.

Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:563]: I'll rebase this PR after #6498 is merged.

**Review by dayo09 — CHANGES_REQUESTED** [review]
You can rebase now :-D

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]
LGTM
